docs/docs/starlark-reference/service-config.md

```diff
@@ -94,7 +94,22 @@ config = ServiceConfig(
     # This field can be used to check the service's readiness after the service has started,
     # to confirm that it is ready to receive connections and traffic
     # OPTIONAL (Default: no ready conditions)
-    ready_conditions = ReadyCondition(...)
+    ready_conditions = ReadyCondition(...),
+
+    # This field is used to specify custom labels at the container level in Docker and Pod level in Kubernetes.
+    # For Docker, the label syntax and format will follow: "com.kurtosistech.custom.key": "value"
+    # For Kubernetes, the label syntax & format will follow: kurtosistech.com.custom/key=value
+
+    # Labels must follow the label standards outlined in [RFC-1035](https://datatracker.ietf.org/doc/html/rfc1035), 
+    # meaning that both the label key and label value must contain at most 63 characters, contain only lowercase 
```

Comment:
Does the custom prefix (e.g. `kurtosistech.com.custom/` or `"com.kurtosistech.custom`) count towards the 63 character limit?

Reply:
it doesn't count for the limit. The 63 max chars limit is for the 'name' part, the prefix has a 256 limit. More here: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set

Reply:
Curious - why do we need the Kurtosis prefix at all on the custom label?

Reply:
well, it wasn't a strong decision, but we wanted to add it in case we want to get all the pods that have custom labels. It's a way to identify the label was created by us and not by the users